docs/CONTRIBUTING.md

```diff
@@ -121,49 +121,6 @@ Kotlin code handles scraping and querying data. Read the [documentation](https:/
 # Install Kotlin.
 brew install kotlin
 ```
-## Project Overview
-The following ASCII file structure diagram shows the most important packages and files, with comments.
```

Comment:
lol this is probably a good idea. We can always generate it later but it's probably daunting seeing such a large blob of information, especially since it's presented as an "overview", as if it clarifies something.